spec/requests/requests_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'managing topics', :type => :request do
+
+  let!(:user) {FactoryBot.create :user}
+
+  it 'redirects unauthenticated users' do
+    post "/topics", :params => { :topic => {:name => "here is a new topic"} }
+    expect(response).to redirect_to('/users/sign_in')
+  end
+
+  it 'allows topic posting for logged in users' do
+    post user_session_path, :params => {'user[email]' => user.email, 'user[password]' => user.password }
+    get '/topics/new'
+    expect(response).to render_template(:new)
```

Comment:
I'd just expect the response *not* to be a redirect to the new_session_path. The internals of where users get sent/redirected to when they are properly authenticated and authorised is covered by feature/system specs.